### PR TITLE
Add draggable compatibility for Jazzmin

### PR DESCRIPTION
**Pull request by doolijb**
Added minimal compatibility for Jazzmin theme. This restores the draggable icon to the stacked inline header. There's still no support for collapsing inlines. I might return to this sometime down the road.

### CODE REVIEW

File: content_editor/static/content_editor/content_editor.js [hunk 1]
```diff
@@ -217,7 +217,7 @@ django.jQuery(function ($) {
       }
     })
 
-    arg.find(">h3").attr("draggable", true)
+    arg.find("h3").attr("draggable", true)
```
Comment (matthiask) [inline content_editor/static/content_editor/content_editor.js:220]: Is there a jazzmin-specific way to find the correct title and not just process all `h3` elements which may exist within the fieldset? We could write `arg.find(">h3, .card-title>h3")` or something then?

File: content_editor/static/content_editor/content_editor.css [hunk 2]
```diff
@@ -107,6 +107,16 @@ h3[draggable] {
...
+/* JAZZMIN DRAGGABLE */
+.order-machine .card-header > .card-title[draggable]::before {
+  content: "drag_indicator";
+  font-family: "Material Icons";
+  font-size: 24px;
+  position: relative;
+  top: 6px;
+  left: -2px;
+}
+
```
Comment (matthiask) [inline content_editor/static/content_editor/content_editor.css:119]: I'm happy with shipping this but I'd prefer it if you could either add it at the end of `content_editor.css` or modify the comments to say `/* JAZZMIN overrides start */` ... `/* JAZZMIN overrides end */` or something to this effect.

**Review by matthiask — COMMENTED** [review]
Thanks! Great to hear that everything mostly just works with jazzmin.

File: content_editor/static/content_editor/content_editor.js (hunk 1 above)
Comment (doolijb) [reply content_editor/static/content_editor/content_editor.js:220]: Sure, give me a day or two and I'll make some updates.

Comment (doolijb) [reply content_editor/static/content_editor/content_editor.js]: Done. Sorry, looks like my formatter freaked out with the semi-colons

**Comment by matthiask** [comment]
Great, thanks! I squashed all your changes and ran `pre-commit` once. I'm now preparing a release.

**Pull request closed without merging**